sq/tests/sdkms/extract_sdkms_import_gpg.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+
+# Parse input flags
+case "$1" in
+    --p256) cipher_suite="nistp256";;
+    --p384) cipher_suite="nistp384";;
+    --p521) cipher_suite="nistp521";;
+    --cv25519) cipher_suite="cv25519";;
+    --rsa3k) cipher_suite="rsa3k";;
+    -*) echo "unknown option: $1" >&2; exit 1;;
```

Comment:
if it's only for test, please ignore it. We may combine the input checkings, add checking for verbosity and failed checking will print usage.

Reply:
What do you mean? Are you suggesting to fix one cipher suite only, for tests?

Reply:
we can add more checking to the verbosity number is in range instead of allowing the program to continue when I type a 3. And we can check number first, then check the first argument, or it will print `unknown option: ` when we don't provide any arguments.

But since it's test, so it seems not important.

```
usage () {
    echo "Usage: extract_sdkms_import_gpg.sh -v <int verbosity [0, 1, 2]>"
}

if (( $# != 3 )); then
    usage
    exit 1
else
    if cipher_suite_is_not_valid; then
       usage
       exit 1
    fi
   
    if number_is_not_valid; then
       usage
       exit 1
    fi
fi
```

Reply:
It is only for testing, but good suggestion

Reply:
- [ ] check verbosity parameter